src/components/SetCampaignContentFromTemplate.tsx

```diff
@@ -32,6 +35,10 @@ export const SetCampaignContentFromTemplate = () => {
     window.console.error("Error creating campaign content from template", data);
   }
 
-  const campaignUrl = `/campaigns/${idCampaign}${search}`;
-  return <Navigate to={campaignUrl} replace={true} />;
+  if (continueUrl) {
+    return <NavigateToExternalUrl to={continueUrl} />;
+  } else {
+    const campaignUrl = `/campaigns/${idCampaign}${search}`;
+    return <Navigate to={campaignUrl} replace={true} />;
+  }
```

Comment:
I do not like this `if`.

I can improve it if `continueUrl` returns `/campaigns/${idCampaign}${search}` in place of _undefinde_ when `continue` query-string parameter is not set.

I have also to create a smart component that based on the URL decides if has to navigate to an external URL or to an internal one.

@Cromeror what do you think?

@fgchaio you have been working on something related, what is your opinion?

Reply:
lgtm

Reply:
Done!